### ISSUE TITLE
Provide Occurrences-API

### ISSUE THREAD

**Issue by nichtich** [hbz/lobid-resources#1407]
As part of coli-conc we created an [Occurrences-API](https://coli-conc.gbv.de/occurrences/) to query occurrences and co-occurrences of identified subjects in K10plus. For simple occurrences the API serves one use case:

> given a subject, how many resources about it does the catalog include and how can I deep link to the result set of this resources?

It's just one query parameter `member` with URI if the SKOS concept (e.g. `https://d-nb.info/gnd/2015876-2`).

In lobid API this would need to be translated to http://lobid.org/resources/search?q=subject.id:https\:\/\/d\-nb.info\/gnd\/2015876\-2%20OR%20subject.componentList.id:https\:\/\/d\-nb.info\/gnd\/2015876\-2&format=json&size=0 to get the `totalItems` and add boilerplate with information about lobid-resources and a deep link into the catalog:

~~~json
{
  "database": {
    "uri": "http://lobid.org/resources/dataset#!",
    "prefLabel": { "de": "lobid-resources – Der hbz-Verbundkatalog als Linked Open Data" },
    "url": "http://lobid.org/resources/"
  },
  "memberSet": [  { "uri": "https://d-nb.info/gnd/2015876-2" } ]
  "count": 52,
  "url": "http://lobid.org/resources/search?q=subject.id:https\\:\\/\\/d\\-nb.info\\/gnd\\/2015876\\-2%20OR%20subject.componentList.id:https\\:\\/\\/d\\-nb.info\\/gnd\\/2015876\\-2"
}
~~~

Sure this is no rocket science, could be done in the client as well.

**Comment by acka47** [comment]
We won't be able to implement this in the coming months. Unassigning for now.

**Comment by TobiasNx** [comment]
@nichtich is this still needed?

**Comment by nichtich** [comment]
Yes, having this would simplify integration into Cocoda and the subject indexing application we are working on. Sure we can work without but then unlikely link to lobid.
